platform/pwl.c

```diff
@@ -9,6 +9,36 @@
 
 #include <errno.h>
 
+/* for mmap */
+#include <sys/mman.h>
+
+#include <gio/gio.h>
+
+// TODO: #if defined(WPE_CHECK_VERSION) && WPE_CHECK_VERSION(1, 3, 0)
```

Comment:
We dont want WPE specific definitions here.  probably the  DEVICE_SCALING should be defined with CMake